backend/backend/common/message.py

```diff
@@ -0,0 +1,63 @@
+import json
+from typing import Protocol
+from datetime import datetime
+from dataclasses import dataclass
+# This module describes how we want a message will appear on our websocket.
+
+VERSION = '1'
+
+# Once a more standard content will be made,
+# every content-type will inherit from abstract class.
+
+
+class Content(Protocol):
+    def serializable(self) -> dict: ...
```

Comment:
I'm very much conflicted, I think it should be BaseModel as it would simplify the need to add "serialized" to pretty much every dataclass - since you simply repeating yourself here....

Each time "serialized" is present you simply copy the __dict__ attribute but more "formaly", however it makes much more sense when you actually thinking about how it might work with websockets

Reply:
done